gpu4pyscf/scf/hf.py

```diff
@@ -419,6 +418,11 @@ def _kernel(mf, conv_tol=1e-10, conv_tol_grad=None,
         _, mf_diis.Corth = mf.eig(fock, s1e)
     else:
         mf_diis = None
+
+    if dump_chk and mf.chkfile:
```

Comment:
`dump_chk` can be disabled by default. dumping chkfile can noticeably slow down the scf function.

Reply:
I think it's a better idea to set `self.chkfile = None` at __init__ function of SCF class (which requires a copy of __init__ function from pyscf.hf.SCF to gpu4pyscf.hf.SCF). If we change the function signature of this internal kernel function, it'll be very hard to turn in on from pysisyphus side.

Reply:
All work. This flag just needs to be muted by default.